apiserver/logsink/filewriter.go

```diff
@@ -7,8 +7,11 @@ import (
 	"io"
 	"os"
 
+	"github.com/juju/juju/service/systemd"
```

Comment:
These two juju/juju imports should be in a block together after the juju/errors and lumberjack imports.